test/Microsoft.TemplateEngine.TemplateLocalizer.EndToEndTests/ExportCommandTests.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.TemplateEngine.TemplateLocalizer.EndToEndTests
+{
+    [TestClass]
+    public class ExportCommandTests
+    {
+        private const string ComplexTemplateJson = @"{
```

Comment:
does it make sense to utilize `TestTemplates` instead?

Reply:
That's possible as well. But I also like how these test are isolated. Let me know if you feel strongly about this. If not, I prefer to keep it.

Reply:
I'm not strong about it, but I see that this template can evolve in the following scenario:
- run the localizer
- replace some strings 
- run the template to see if it's localized

Also it's very easy to produce non-runnable template.json this way as it is not validated in anyway and schema is not helping you to write it.  It would be also good to run it before/after localization with dotnet new to see if it's not broken - testing malformed json might not do valid test.

Reply:
please consider at least move it to separate file to enable schema completion when editing.

Reply:
Running dotnet new to see if the files can be read can only be done after https://github.com/dotnet/templating/pull/3089 is merged. I still changed it to use the test templates project for the other reasons you listed.
If schema changes, we will update the test templates. I doubt anyone will remember to update the const strings. So it's better to use the test templates it seems.